astropy/wcs/src/astropy_wcs.c

```diff
@@ -28,6 +31,24 @@ static PyTypeObject WcsType;
 static int _setup_wcs_type(PyObject* m);
 
 
+PyObject* PyWcsprm_set_wtbarr_fitsio_callback(PyObject *dummy, PyObject *args) {
+    PyObject *callback;
+
+    if (PyArg_ParseTuple(args, "O:set_wtbarr_fitsio_callback", &callback)) {
+        if (!PyCallable_Check(callback)) {
+            PyErr_SetString(PyExc_TypeError, "parameter must be callable");
+            return NULL;
+        }
+        _set_wtbarr_callback(callback);
+
+        /* Boilerplate to return "None" */
+        Py_INCREF(Py_None);
```

Comment:
There's the [`Py_RETURN_NONE`](https://docs.python.org/3/c-api/none.html#c.Py_RETURN_NONE) macro. Also the comment above isn't necessary.